src/styles/blocks/page.scss

```diff
@@ -0,0 +1,10 @@
+.page {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+}
+
+.page__body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
```

Comment:
The 'margin: 0;' rule in '.page__body' is used to reset the default margin of the body. However, it's a common practice to have a separate reset or normalize file where you reset margins, paddings, and other styles for all elements to have more control and consistency across different browsers. This is a code quality recommendation.